packages/dev/core/src/FlowGraph/flowGraphPath.ts

```diff
@@ -0,0 +1,155 @@
+import { RegisterClass } from "../Misc/typeStore";
+import type { FlowGraphContext } from "./flowGraphContext";
+
+// Path: /x/{y}/z/.../w
+const PATH_REGEX = /[./]({?\w+}?)/g;

```

Comment:
Does it really work? Shouldn't the `{` and `}` characters be escaped, as they are special characters in a regexp? Maybe because they are followed by `?` you don't have to escape them...

Reply:
Yup, it does! You can quickly check on console by doing `[..."/x/{y}/z".matchAll(/[./]({?\w+}?)/g)]`.

Reply:
In fact, I tested and it did work, but I didn't understand why, that's why I left the comment :)